pghoard/config.py

```diff
@@ -58,3 +61,33 @@ def set_config_defaults(config, *, check_commands=True):
         site_config.setdefault("stream_compression", False)
 
     return config
+
+
+def read_json_config_file(filename, *, check_commands=True):
+    try:
+        with open(filename, "r") as fp:
+            config = json.load(fp)
+    except FileNotFoundError:
+        raise InvalidConfigurationError("Configuration file {!r} does not exist".format(filename))
+    except ValueError:
+        raise InvalidConfigurationError("Configuration file {!r} does not contain valid JSON".format(filename))
```

Comment:
add the str() of the ex to the error message, JSON missing commas etc. may be hard to spot otherwise